### PR TITLE
bpo-42931: Add randbytes to random.__all__

### PR DESCRIPTION
**Pull request by sbalian**
https://bugs.python.org/issue42931

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@sbalian

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by rhettinger — APPROVED** [review]

**Comment by sbalian** [comment]
@rhettinger Thanks, I've signed it.

File: Misc/NEWS.d/next/Library/2021-01-15-00-23-50.bpo-42931.QD6U2B.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add randbytes to random.__all__
```
Comment (vstinner) [inline Misc/NEWS.d/next/Library/2021-01-15-00-23-50.bpo-42931.QD6U2B.rst]: Formating suggestion:

```suggestion
Add :func:`randbytes` to ``random.__all__``.
```

**Comment by vstinner** [comment]
Thanks for the fix @sbalian ;-)

**Pull request merged**